test/integration/targets/copy/tasks/tests.yml

```diff
@@ -1488,3 +1488,594 @@
     - 'new_sub_dir1/foo.txt'
   loop_control:
     loop_var: 'dest'
+
+#
+# Recursive copying on remote host
+#
+## prepare for test
+- block:
+
+  - name: execute - create a user for test
+    user:
+      name: ptux
+      uid: 14131
+      state: present
+
+  - name: execute - create a group for test
+    group:
+      name: ptux
+      gid: 14131
+      state: present
```

Comment:
@mattclay do we already have a standard user and group for testing setting file ownership  (nobody maybe?)

Reply:
No, we don't. If the test needs additional user(s) it should create them in a `block` and use `always` to remove them. The test should also have the `needs/root` alias.

Reply:
I was thinking the test is executed in a container.
Everything will go when the test over. Am I wrong?

Also, What does [needs/root alias] mean?

Reply:
Our CI system executes the tests in a container. However, we support running them in other environments and do not require the use of containers, so tests should clean up after themselves.

Reply:
got it.

Reply:
Use a variable for the user and group and set them to a descriptive prefix like "ansible_copy_test_" and a random number. That will nearly eliminate any possibility of it matching an existing user, while making it more obvious where it came from if still present when the test exists (ex: test traceback).